package.json

```diff
@@ -13,10 +13,10 @@
   "scripts": {
     "test": "nyc mocha src/tests/*.js --require @babel/register",
     "coverage": "nyc report --reporter=text-lcov | coveralls",
-    "start": "node dist/index.js",
+    "start": "node -r dotenv/config dist/index.js",
```

Comment:
what does this `-r dotenv/config` do?

Reply:
This load the variables from .env into process.env, so there is no need to import dotenv on every file it is needed